tests/SprykerEcoTest/Service/Braintree/BraintreeServiceTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Service\Braintree;
+
+use Codeception\Test\Unit;
+use SprykerEco\Service\Braintree\BraintreeService;
+use SprykerEco\Service\Braintree\BraintreeServiceFactory;
+use SprykerEco\Service\Braintree\BraintreeServiceInterface;
+use SprykerEco\Service\Braintree\Model\TokenGenerator\TokenGeneratorInterface;
+
+/**
+ * @group SprykerEcoTest
+ * @group Service
+ * @group Braintree
+ * @group BraintreeTest
+ * @group BraintreeServiceTest
+ */
+class BraintreeServiceTest extends Unit
+{
+    protected static $tokenValue;
+
+    /**
+     * @return void
+     */
+    public function testGenerateToken(): void
+    {
+        $service = $this->prepareService();
+
+        $this->assertNotEmpty($service->generateToken());
```

Comment:
don't sure if we need this test that way. It's mock, so we don't test actual token generation